pyairtable/utils.py

```diff
@@ -21,7 +23,9 @@ def datetime_from_iso_str(value: str) -> datetime:
     Args:
         value: datetime string e.g. "2014-09-05T07:00:00.000Z"
     """
-    return datetime.strptime(value, "%Y-%m-%dT%H:%M:%S.%fZ")
+    if "Z" in value:
```

Comment:
Shouldnt we rather check if it and with z? Because from what I understand basically the delimiter could also be a Z, even though that might be never used. But then the parsing would fail anyways, I dont know. Can't we even parse the Z with the %z?